bin/run_tests

```diff
@@ -288,6 +291,19 @@ test_deployment() {
     run_docker_test systemd_services -s integration_test
 }
 
+test_xo() {
```

Comment:
If changes are made to the whole xo family (for instance, changing the namespace), it's convenient to be able to run all the xo tests at once.

The main `run_tests` module list, however, doesn't include the xo module:
```
MODULE_LIST="
    signing
    cli
    rest_api
    poet
    config
    validator
    go_sdk
    java_sdk
    javascript_sdk
    python_sdk
    integration
"
```
Someone working on xo can run `run_tests -m xo`, but nobody else needs to worry about it. The PoET unit tests can already be run in this way.